modules/acquia_cms_search/config/install/views.view.search.yml

```diff
@@ -326,21 +328,21 @@ display:
       tags:
         - 'config:search_api.index.content'
   search:
-    display_plugin: page
     id: search
     display_title: Page
+    display_plugin: page
     position: 1
     display_options:
-      display_extenders: {  }
-      path: search
       cache:
         type: search_api_tag
         options: {  }
       defaults:
         cache: false
       exposed_block: true
+      display_extenders: {  }
+      path: search
     cache_metadata:
-      max-age: -1
+      max-age: 0
```

Comment:
@apathak18 Any reason for setting max-age as 0. This will makes the view block/page un cacheable. Is that intended behaviour ?

Reply:
Yes @vishalkhode1 , if we revert this then error remains.